app/controllers/backoffice/customers_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Backoffice
+  class CustomersController < ApplicationController
+    include AdminsOnly
+
+    def index
+      @statuses = Customer.statuses
+
+      @customers = if params[:status].present? && params[:query].present?
```

Comment:
cara se tiver usando o ranksack, acho que nao é um if q se faz nao. mas uma hash de queries. pq pode-se usar mais de um filtro por vez. passa-se a hash pro ransack. se nao for isso. perdoe-me @emilyfaccin